tests/test_arrow.py

```diff
@@ -74,9 +74,6 @@ def test_init_with_fold(self):
         before = arrow.Arrow(2017, 10, 29, 2, 0, tzinfo="Europe/Stockholm")
         after = arrow.Arrow(2017, 10, 29, 2, 0, tzinfo="Europe/Stockholm", fold=1)
 
-        assert hasattr(before, "fold")
-        assert hasattr(after, "fold")
```

Comment:
This basically checks for presence of the `Arrow.fold` property, seems unnecessary.